source/content/guides/composer-convert.md

```diff
@@ -3,215 +3,38 @@ title: Convert a Standard Drupal 8 Site to a Composer Managed Site
-
-```bash{promptUser:user}
-cp ../$site/pantheon.upstream.yml .
-```
-
-`ls` should reveal that the new code repository now has a copy of the `pantheon.upstream.yml`.
-
-## Add in the Custom and Contrib Code Needed to Run Your Site
-
-What makes your site code unique is your selection of contributed modules and themes, and any custom modules or themes your development team has created. These customizations need to be replicated in your new project structure.
-
-### Contributed Code
-
-#### Modules
-
-A Composer-managed site should be able to include all custom code via Composer. Begin by reviewing your existing site's code. Check for contributed modules in `/modules`, `/modules/contrib`, `/sites/all/modules`, and `/sites/all/modules/contrib`.
-
-When reviewing your site, take stock of exactly what versions of modules you depend on. One way to do this is to use a command like the following from within a contributed modules folder (e.g. `/modules`, etc):
-
-```bash{outputLines:2-4}
-find . -maxdepth 2 -name '*.info.yml' \
-  -exec basename '{}' '.info.yml' ';' \
-  -exec grep -e '^version' '{}' ';' \
-  -exec echo ';'
-```
-
-This will list each module followed by the version of that module that is installed.
-
-You can add these modules to your new codebase using Composer by running the following for each module in the `$site-composer` directory:
-
-```{promptUser:user}
-composer require drupal/MODULE_NAME:^VERSION
-```
-
-Where `MODULE_NAME` is the machine name of the module in question, and `VERSION` is the version of that module your site is currently using. Composer may pull in a newer version than what you specify, depending upon what versions are available. You can read more about the caret (`^`) [in the documentation for Composer](https://getcomposer.org/doc/articles/versions.md#caret-version-range-) itself.
-
-If you get the following error:
-
-```none
-[InvalidArgumentException]
-Could not find a version of package drupal/MODULE_NAME matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.
-```
+<Alert title="Note" type="info">
 
-It means that one of the modules you are using (or its dependencies) are not stable. If there is not a stable version you can switch to, you may need to adjust the `minimum-stability` setting of `composer.json` to a more relaxed value, such as `beta`, `alpha`, or even `dev`. You can read more about `minimum-stability` [in the documentation for Composer](https://getcomposer.org/doc/04-schema.md#minimum-stability) itself.
+To maintain best practice and to avoid difficult, time-consuming repairs to the site, this doc is written for users with access to Pantheon's [Multidev](/multidev) feature.
 
-#### Themes
+Pantheon support is not available to users who avoid the Multidev steps.
```

Comment:
We can direct people to the "Manually Migrate" page (or the renamed version thereof) to follow the "create a new site" path.